datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -202,7 +204,7 @@ def add_categories(self, categories):
                 raise NotImplementedError()
             self.categories[ann_type.name] = converted_desc
 
-    def write(self):
+    def write(self, pool: Optional[Pool] = None):
```

Comment:
How about just accepting all arguments since `_SubsetWriter` could parents of other than one from `datumaro_binary`?
```suggestion
    def write(self, *args, **kwargs):
```

Reply:
Same here.